src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -1,11 +1,33 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToOne;
+import java.util.ArrayList;
+import java.util.List;
 
-@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity(name = "carts")
```

Comment:
Czemu tu są jakiekolwiek zmiany? Wszystko przez to że nie masz merge aktualnego main....